core/spec/models/spree/zone_spec.rb

```diff
@@ -302,4 +302,29 @@
       end
     end
   end
+
+  context "state and country associations" do
+    let!(:country)  { create(:country) }
+
+    context "has countries associated" do
+      let!(:zone) do
+        create(:zone, countries: [country])
+      end
+
+      it "can access associated countries" do
+        expect(zone.countries).to include(country)
```

Comment:
Could this be `expect(zone.countries.to_a).to eq([country])`? Just to make the expectation a little stricter?